ThetasketchEstimate not working on Python2.7

When unit tests are run on Python2.7 

TestPostAggregators.test_build_thetapostaggregator fails with 

```
E       AssertionError: assert [{'field': <p...tchEstimate'}] == [{'field': {'f...tchEstimate'}]
E         At index 0 diff: {'field': <pydruid.utils.postaggregator.ThetaSketch instance at 0x1116e9ea8>, 'type': 'thetaSketchEstimate', 'name': 'pag1'} != {'field': {'fieldName': 'theta1', 'type': 'fieldAccess'}, 'type': 'thetaSketchEstimate', 'name': 'pag1'}
```

It looks like the object is being used in the dictionary rather than the object's post_aggregator. Will drop a PR to fix.

--- a/pydruid/utils/postaggregator.py
+++ b/pydruid/utils/postaggregator.py
@@ -152,7 +152,7 @@ class LongLeast(Postaggregator):
                 'type': 'longLeast', 'name': name, 'fields': [f.post_aggregator for f in fields]}
 
 
-class ThetaSketchOp:
+class ThetaSketchOp(object):
     def __init__(self, fn, fields, name):
         self.post_aggregator = {'type': 'thetaSketchSetOp',
                                 'name': name,

